src/archivist/recorder/repositories/git/git.js

```diff
@@ -120,6 +119,12 @@ export default class Git {
     return this.git.raw([ 'restore', '-s', commit, '--', path ]);
   }
 
+  async cleanUp() {
```

Comment:
it now seems weird to me that

`cleanUp` function is using `this.git.clear`
`clear` function is using `this.git.clean`

Maybe we should harmonize this